tests/proc-test/src/operations.test.ts

```diff
@@ -0,0 +1,174 @@
+      DROP ROUTINE IF EXISTS "StoredProcTest_helloProcedureLocal_f";`);
+  });
+
+  afterAll(async () => {
+    await testRuntime?.destroy();
+  });
+
+  test("test-procReplay", async () => {
+    const now = Date.now();
+    const user = `procReplay-${now}`;
+    const res = await testRuntime.invoke(StoredProcTest).helloProcedure(user);
+    expect(res).toMatch(`Hello, ${user}! You have been greeted 1 times.`);
+
+    const wfUUID = `procReplay-wfid-${now}`;
+    expect(await testRuntime.invoke(StoredProcTest, wfUUID).getGreetCount(user)).toBe(1);
+
+    await testRuntime.retrieveWorkflow(wfUUID).getResult();
+
+    const statuses = await runSql({ ...config.poolConfig, database: config.system_database }, async (client) => {
+      const { rows } = await client.query<workflow_status>("SELECT * FROM dbos.workflow_status WHERE workflow_uuid = $1", [wfUUID]);
+      return rows;
+    });
+
+    expect(statuses.length).toBe(1);
+    expect(statuses[0].status).toBe("SUCCESS");
+    expect(statuses[0].output).toBe(JSON.stringify(1));
+    expectNullResult(statuses[0].error);
+
+    expect(await testRuntime.invoke(StoredProcTest, wfUUID).getGreetCount(user)).toBe(1);
+
+    const txRows = await testRuntime.queryUserDB<transaction_outputs>("SELECT * FROM dbos.transaction_outputs WHERE workflow_uuid=$1", wfUUID);
+    expect(txRows.length).toBe(1);
+    expect(txRows[0].function_id).toBe(0);
+    expect(txRows[0].output).toBe("1");
+    expectNullResult(txRows[0].error);
+  });
+
+  test("test-procGreetingWorkflow", async () => {
+    const wfUUID = uuidv1();
+    const user = "procWF";
+    const res = await testRuntime.invokeWorkflow(StoredProcTest, wfUUID).procGreetingWorkflow(user);
+    expect(res.count).toBe(0);
+    expect(res.greeting).toMatch(`Hello, ${user}! You have been greeted 1 times.`);
+
+    const txRows = await testRuntime.queryUserDB<transaction_outputs>("SELECT * FROM dbos.transaction_outputs WHERE workflow_uuid=$1", wfUUID);
+    expect(txRows.length).toBe(2);
+    expect(txRows[0].function_id).toBe(0);
+    expect(txRows[0].output).toBe("0");
+    expectNullResult(txRows[0].error);
+
+    expect(txRows[1].function_id).toBe(1);
+    expect(txRows[1].output).toMatch(`Hello, ${user}! You have been greeted 1 times.`);
+    expectNullResult(txRows[1].error);
+  });
+
+  test("test-debug-procGreetingWorkflow", async () => {
+    const wfUUID = uuidv1();
+    const user = "debugProcWF";
+    const res = await testRuntime.invokeWorkflow(StoredProcTest, wfUUID).procGreetingWorkflow(user);
+    expect(res.count).toBe(0);
+    expect(res.greeting).toMatch(`Hello, ${user}! You have been greeted 1 times.`);
+
+    const debugConfig = { ...config, debugMode: true };
+    const debugRuntime = await createInternalTestRuntime([StoredProcTest], debugConfig);
+    try {
+      expect(debugRuntime).toBeDefined();
+      const debugRes = await debugRuntime.invokeWorkflow(StoredProcTest, wfUUID).procGreetingWorkflow(user);
```

Comment:
To make sure we can correctly find the recorded name by workflow UUID, we can add a line to test `executeWorkflowUUID` similar to https://github.com/dbos-inc/dbos-transact/blob/main/tests/testing/debugger.test.ts#L160-L162

This makes sure we're able to recover a stored proc (correctly find function name, class name, and proc name) when we just provide a WF UUID.